hawkular-alerts-bus/src/main/java/org/hawkular/alerts/bus/listener/AvailDataListener.java

```diff
@@ -115,7 +115,7 @@ protected void onBasicMessage(AvailDataMessage msg) {
     protected String convertReceivedMessageClassNameToDesiredMessageClassName(String className) {
 
         if (className.equals("org.hawkular.metrics.component.publish.AvailDataMessage")) {
-            return "org.hawkular.alerts.bus.messages.AvailDataMessage";
+            return "org.hawkular.alerts.bus.api.AvailDataMessage";
```

Comment:
yes .class.getCanonicalName(), but constant is also ok. It was just and idea that could avoid previous error. 

https://github.com/hawkular/hawkular-datamining/blob/inventory-direct-query/hawkular-datamining-bus/src/main/java/org/hawkular/datamining/bus/listener/MetricDataListener.java#L93